### ISSUE TITLE
Love to see some pics!

### ISSUE THREAD

**Issue by ghost** [numidium/wcngine#1]
I found you from DFCommunity, I don't have time today to download and compile but would love to see a preview pic or what it looks like or what you want for it to look like. Love the idea.

**Comment by numidium** [comment]
Hello, @communityus! Unfortunately I don't have screenshots of this yet. I was mainly experimenting with ways to draw space with six degrees of freedom in fake 3D. I kinda gave up after a while in favor of drawing flat planes with OpenGL. Never really got far with that though. This would probably be a worthwhile project to pick back up now that I think about it.

**Comment by ghost** [comment]
I see. So kinda like a CPU based 3d renderer - but not just for viewing models, for gameplay i.e. space game? Or am I off base?

**Comment by numidium** [comment]
Yep, I intend to make this a full-fledged game engine. At some point I want to reverse-engineer the old Wing Commander (what the "wc" stands for) games and try to port them to this. Kind of ambitious for my skillset but I wanted to give it a shot.

**Comment by ghost** [comment]
Fun. If the approach is anything like DFU it would be amazing!

FYI there is some WC stuff out there mainly:
https://www.wcsaga.com/
- based on hard light engine
https://www.hard-light.net/
and then https://pioneerspacesim.net/

it took me a while just trying to track them down again, so posting in case you never ran across them.
